pipelines/acs_2020_manual_update.py

```diff
@@ -5,54 +5,112 @@
 import pandas as pd
 import re
 
-def parse_args() -> Tuple[str, str, str]:
+OUTPUT_SCHEMA_COLUMNS = ['labs_geotype', 'labs_geoid', 'pff_variable', 'base_variable', 'e', 'm', 'c', 'p', 'z', 'domain']
+
+def parse_args() -> Tuple[str, str]:
     parser = argparse.ArgumentParser()
-    parser.add_argument("-i", "--input", type=str, help="Location of input ACS file")
+
     parser.add_argument(
         "-y", "--year", type=str, help="The ACS5 year, e.g. 2019 (2014-2018)"
     )
     parser.add_argument(
         "-g", "--geography", type=str, help="The geography year, e.g. 2010_to_2020"
     )
     args = parser.parse_args()
-    return args.input, args.year, args.geography
+    return args.year, args.geography
+
+def pivot_field_name(df, field_name, domain):
+    field_name_df = split_by_field_name(df, field_name)
+
+    field_name_df.rename(columns=lambda column_name: re.sub(f"^{ field_name }(E|M|C|P|Z)$",r"\1", column_name).lower(), inplace=True)
+    field_name_df['pff_variable'] = field_name.lower()
+    field_name_df['domain'] = domain
+
+    return field_name_df
+
+def extract_field_names(df):
+    return df.columns[2:].str[:-1].drop_duplicates()
+     
+def split_by_field_name(df, pff_field_name):
+    return df.filter(regex=f"^(GeoType|GeoID|{ pff_field_name }(E|M|C|P|Z))$")
+
+def strip_unnamed_columns(df):
+    return df.loc[:,~df.columns.str.match("Unnamed")]
+
+def sheet_names(year):
+    if year == '2010':
+        sheet_name_suffix = '0610'
+        inflated = "_Inflated"
+    if year == '2020':
+        sheet_name_suffix = '1620'
+        inflated = ""
+
+    domains_sheets = [
```

Comment:
I ran into an issue here where I accidentally entered the year as `2019` instead of `2020`, and got an error that took me a little bit to figure out. I personally like to add assertions that the inputted args are within a list of acceptable values. It helps to give a better error message and makes it easier for other engineers to figure out how to run your code

Reply:
Yep makes sense, was thinking about that